cypress/components/flat-table/flat-table.cy.js

```diff
@@ -2749,7 +2749,9 @@ const FlatTablePagerStickyHeaderComponent = () => {
 };
 
 const FlatTablePartiallySelectedOrHighlightedRows = ({
+  // eslint-disable-next-line react/prop-types
```

Comment:
question (non-blocking): Would it be better to just ignore this rule all together in any cypress (JS) files